cli/internal/cmd/verify.go

```diff
@@ -207,6 +233,139 @@ func addPortIfMissing(endpoint string, defaultPort int) (string, error) {
 	return "", err
 }
 
+// a formatter formats the attestation document.
+type formatter interface {
```

Comment:
Please give this a name like `attestationDocFormatter`. Remember this is in scope of the whole package, not only the file. Implementation can have an Impl suffix if no better naming comes up.